src/app/pages/errata-summary/table/table.tsx

```diff
@@ -131,19 +131,24 @@ const sortFunctions = {
 function DesktopHeaderColumn({colSpec, sortController}: DesktopHeaderColumnProps): React.ReactElement {
     const {sortKey, sortDir, setSortFn, setSortDir, setSortKey} = sortController;
     const sortable = 'sortFn' in colSpec;
-    const onClick = () => {
-        if (sortKey === colSpec.id) {
-            setSortDir(-sortDir);
-        } else {
-            // @ts-expect-error onClick is only used when sortable
-            setSortFn(colSpec.sortFn);
-            setSortKey(colSpec.id);
-            setSortDir(1);
+    const sortAttributes = React.useMemo(() => {
```

Comment:
I changed this to only have onClick defined when sortable was set, which makes sense.